packages/sdk/src/models/decrypted-keypair.ts

```diff
@@ -0,0 +1,50 @@
+import { Keypair as APIKeypair } from '@meeco/keystore-api-sdk';
+import cryppo from '../services/cryppo-service';
+import RSAPrivateKey from './rsa-private-key';
+import RSAPublicKey from './rsa-public-key';
+import { SymmetricKey } from './symmetric-key';
+
+/**
+ * An asymmetric encryption (public/private) key pair, with the private key decrypted.
+ *
+ * Encryption and decryption methods are defined on the classes {@link RSAPublickey} and {@link RSAPrivatekey} which may
+ * be used if only one key is available.
+ */
+export default class DecryptedKeypair {
+  publicKey: RSAPublicKey;
+  privateKey: RSAPrivateKey;
+
+  /** A new random keypair with default length. */
+  static async new(keyBits = 4096): Promise<DecryptedKeypair> {
+    const { privateKey, publicKey } = await cryppo.generateRSAKeyPair(keyBits);
+    return new DecryptedKeypair(publicKey, privateKey);
+  }
+
+  /**
+   * Decrypts a keypair from the Meeco Keystore.
+   * @param masterKey Usually the user's key encryption key.
```

Comment:
Should probably be `keyEncryptionKey` then. `masterKey` generally refers to the passphrase derived key (albeit the name is now deprecated)